demos/apps/sleep/sleep.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
```

Comment:
It doesn't listen on port 80. I just used it as a dummy port, so that the caller would be a proper service, not just a POD, so that we can use it to experiment with rules that are source specific.

Using the echo server would make this more than just a sample source service, which might make it more confusing. Btw, we do also have an echo sample under istio/demos. Is that the same code as in gcr.io/google_containers/echoserver:1.4? If so, maybe that one just needs to be cleaned up and used for both purposes.

Thoughts?